components/concourse-task-toolbox/Dockerfile

```diff
@@ -29,6 +29,7 @@ RUN apk add --update \
 	py3-pip \
 	mailcap \
 	ncurses \
+	gomplate \
```

Comment:
:thinking: mini helm

Reply:
Yeah...couldn't get helm to do this without a whole chart being present (understandably). `gomplate` is not as nice to work with....but it solved the problem. If there are better alternatives, I'm open to suggestions.